CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This file is used to list changes made in each version of the postfix cookbook.
 
 ## Unreleased
 
+- postfix package assumes ipv6 is always available, however this is not always true. In order to tell postmap this is the case the configs need to be written first
```

Comment:
Can you please shorten this? It's a bit long and I think what you used as the subject for this PR would work well.

Reply:
Sure, let me ammend the PR